src/js/components/Ballot/PositionItem.jsx

```diff
@@ -71,22 +65,18 @@ export default class PositionItem extends Component {
             /> :
           image_placeholder }
         <div className="position-item__content">
-            <h4 className="position-item__name">
-              <Link to={speaker_we_vote_id_link}>
-                { this.props.speaker_display_name }
-              </Link>
-              { position.is_support && !position.vote_smart_rating ? <span>
-                  &nbsp;support</span> : null }
-              { position.is_oppose && !position.vote_smart_rating ? <span>
-                  &nbsp;oppose</span> : <span></span> }
-            </h4>
-              { position_description }
+          <h4 className="position-item__display-name">
+            <Link to={speaker_we_vote_id_link}>
+              { this.props.speaker_display_name }
+            </Link>
+          </h4>
+            { position_description }
```

Comment:
The use of `position_description` seems like a holdover from something else. Even if we keep this structure, it seems like it might make sense to rename.